lib/ui/window.dart

```diff
@@ -255,6 +294,16 @@ class Locale {
     }
   }
 
+  /// The script subtag for the locale.
+  ///
+  /// This can be null.
+  ///
+  /// This is expected to be a valid Unicode Language Identifier script subtag
+  /// that is listed in Unicode CLDR supplemental data:
+  /// http://unicode.org/cldr/latest/common/validity/script.xml. Please see
+  /// constructor documentation.
```

Comment:
"http://unicode.org/cldr/latest/common/validity/script.xml." isn't a valid URL. Our usual style would be to use:

```suggestion
   /// The script subtag for the locale.
￼   ///
￼   /// This may be null, indicating that there is no specified script subtag.
￼   ///
￼   /// This should be a valid Unicode Language Identifier script subtag
￼   /// as listed in the [Unicode CLDR supplemental data file](http://unicode.org/cldr/latest/common/validity/script.xml). 
   ///
   /// See also:
   ///
   ///  * [new Locale], which describes the conventions for creating [Locale] objects.
￼```

Reply:
but generally, it's better to put the documentation on the properties and have the constructors point to the properties, than vice versa.